samples/emoji-search/android-views/src/main/java/app/cash/zipline/samples/emojisearch/views/EmojiSearchActivity.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.samples.emojisearch.views
+
+import android.app.Activity
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.compose.runtime.NoLiveLiterals
+import app.cash.redwood.compose.AndroidUiDispatcher.Companion.Main
+import app.cash.redwood.protocol.widget.ProtocolMismatchHandler
+import app.cash.redwood.treehouse.*
```

Comment:
No star imports! We should change spotless to run over these samples...

Reply:
Must be because I re-installed AS.

I usually commit `.idea/codeStyles` in my projects so that I can use the reformat command in Intellij without having to manually set up the styles for each project.